hieradata/common.yaml

```diff
@@ -252,8 +252,8 @@ govuk::apps::content_register::rabbitmq_hosts:
   - rabbitmq-2.backend
   - rabbitmq-3.backend
 
-govuk::apps::content_store::nagios_memory_warning: 800
-govuk::apps::content_store::nagios_memory_critical: 900
+govuk::apps::content_store::nagios_memory_warning: 1000
```

Comment:
Should these be lower, e.g. 900 and 1000 respectively, given the graph shown in the PR description?